<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TuloMenoApuri</class>
 <widget class="QWidget" name="TuloMenoApuri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1239</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <widget class="QTableView" name="tilellaView"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="tiliLabel">
         <property name="text">
          <string>Tili</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="TilinvalintaLine" name="tiliEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="maaraLabel">
         <property name="text">
          <string>Määrä</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KpEuroEdit" name="maaraEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="verotonLabel">
         <property name="text">
          <string>Veroton</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="KpEuroEdit" name="verotonEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="alvLabel">
         <property name="text">
          <string>Alv</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="alvCombo"/>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="alvSpin">
           <property name="suffix">
            <string> %</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="vahennysCheck">
         <property name="text">
          <string>Ei oikeuta alv-vähennykseen</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="kohdennusEdit">
         <property name="text">
          <string>Kohdennus</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="kohdennusCombo"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="merkkauksetLabel">
         <property name="text">
          <string>Merkkaukset</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="merkkauksetEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="jaksotusLabel">
         <property name="text">
          <string>Jaksotus</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="KpDateEdit" name="alkuEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="jaksoViiva">
           <property name="text">
            <string> - </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KpDateEdit" name="loppuEdit"/>
         </item>
        </layout>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="poistoLabel">
         <property name="text">
          <string>Poistoaika</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QSpinBox" name="poistoSpin">
         <property name="suffix">
          <string> vuotta</string>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="seliteLabel">
         <property name="text">
          <string>Selite</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="seliteEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Lisää rivi</string>
         </property>
         <property name="icon">
          <iconset resource="../pic/pic.qrc">
           <normaloff>:/pic/lisaarivi.png</normaloff>:/pic/lisaarivi.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Poista rivi</string>
         </property>
         <property name="icon">
          <iconset resource="../pic/pic.qrc">
           <normaloff>:/pic/poistarivi.png</normaloff>:/pic/poistarivi.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="maksutapaLabel">
         <property name="text">
          <string>Maksutapa</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="maksutapaCombo">
         <item>
          <property name="text">
           <string>Lasku</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/lasku.png</normaloff>:/pic/lasku.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pankkitili</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/pankki.png</normaloff>:/pic/pankki.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Luottokortti</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/luottokortti.png</normaloff>:/pic/luottokortti.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Käteinen</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/kateinen.png</normaloff>:/pic/kateinen.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hyvitys</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/poista.png</normaloff>:/pic/poista.png</iconset>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="toimittajaLabel">
         <property name="text">
          <string>Toimittaja</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLineEdit" name="toimittajaEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="toimittajaButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../pic/pic.qrc">
             <normaloff>:/pic/yrittaja.png</normaloff>:/pic/yrittaja.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Viite</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="viiteEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="erapaivaLabel">
         <property name="text">
          <string>Eräpäivä</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="KpDateEdit" name="erapaivaEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="vastatiliLabel">
         <property name="text">
          <string>Vastatili</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="TilinvalintaLine" name="vastatiliEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="eraLabel">
         <property name="text">
          <string>Tase-erä</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="eraCombo"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TilinvalintaLine</class>
   <extends>QLineEdit</extends>
   <header location="global">db/tilinvalintaline.h</header>
  </customwidget>
  <customwidget>
   <class>KpEuroEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">tools/kpeuroedit.h</header>
  </customwidget>
  <customwidget>
   <class>KpDateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">tools/kpdateedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections/>
</ui>
